apps/fxlab/app/src/main/res/layout/activity_main.xml

```diff
@@ -22,13 +22,34 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+	<com.google.android.material.appbar.AppBarLayout
+		android:id="@+id/appBar"
```

Comment:
more tabs!

Reply:
I know - so annoying!! I have everything in Android Studio checked to use spaces. I don't know how they keep creeping in!

Reply:
I think they're fixed now.